pkg/minikube/pause/pause.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pause
+
+import (
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"k8s.io/minikube/pkg/minikube/command"
+	"k8s.io/minikube/pkg/minikube/cruntime"
+	"k8s.io/minikube/pkg/minikube/kubelet"
+)
+
+// Pause pauses a Kubernetes cluster
+func Pause(cr cruntime.Manager, r command.Runner) error {
```

Comment:
What do you think about moving it either to cruntime or make a new package for all actions
such as ops.Start, ops.Stop, ops.Pause,...

Reply:
Good idea. I think this is the intent of the `cluster` class, so we should probably just use that.